### DISCUSSION TITLE
Issue while downloading

### DISCUSSION THREAD

**Discussion by dhruvsolanki2188** [community/community#65741] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![Screenshot_20230902-130653](https://github.com/community/community/assets/86949033/2405a9b6-df4f-4340-a610-375b1b32133f)
Kindly check it out

**Comment by denverharry** [comment]
I am also facing downloading issue on my [Bristol concrete company](https://www.bristolconcretecontractors.com/) website. I am trying to download images from this website. If you get help please also guide me.

**Comment by dhruvsolanki2188** [comment]
For sure




With regards
Dhruv solanki
